pkg/api/bzz.go

```diff
@@ -355,14 +355,10 @@ FETCH:
 				dirPath := pathVar + "/"
 				exists, err := m.HasPrefix(ctx, dirPath)
 				if err == nil && exists {
-					// redirect to directory
```

Comment:
This is a feature being disabled. The problem mentioned in the issue is the redirect loop and not the redirect. First, we need to understand how this feature is being used. I think the way this is used is that it would redirect you to the folder in case you entered an incorrect path. This way it would hit the `/bzz` with empty path and this would enter the previous if loop on L330.